src/auth.js

```diff
@@ -0,0 +1,95 @@
+import auth0 from 'auth0-js'
+import Vue from 'vue'
+
+// exchange the object with your own from the setup step above.
+let webAuth = new auth0.WebAuth({
+  domain: 'tym-oao.auth0.com',
+  clientID: '3WW4lg3F0kOBxxRABPvhcW3eGJMKrMIu',
+  // TODO: get redirectUri from environment variable
+  // redirectUri: 'http://localhost:8080/callback',
+  redirectUri: 'https://nerium-static.ty-m.pw/callback',
+  // we will use the api/v2/ to access the user information as payload
+  audience: 'https://tym-oao.auth0.com/api/v2/',
+  responseType: 'token id_token',
+  scope: 'openid profile'
+})
+
+let auth = new Vue({
+  computed: {
+    token: {
+      get: function() {
+        return localStorage.getItem('id_token')
```

Comment:
I've always been a fan of local storage. 😄 👍 Very nice use of it in this file. ✨

Reply:
Yeah, most Auth0 docs and examples suggest using local storage for keeping their credential payloads. Vuex might also be an option here, but I would say only as this iterates up to more complex applications.